keras/src/quantizers/__init__.py

```diff
@@ -13,7 +17,14 @@
 from keras.src.saving import serialization_lib
 from keras.src.utils.naming import to_snake_case
 
-ALL_OBJECTS = {Quantizer, AbsMaxQuantizer}
+ALL_OBJECTS = {
+    Quantizer,
+    AbsMaxQuantizer,
+    QuantizationConfig,
+    Int8QuantizationConfig,
+    Int4QuantizationConfig,
+    Float8QuantizationConfig,
```

Comment:
This is now mixing quantizers and quantization configs. That seems weird.

The truth is that `serialize` and `deserialize` are pretty useless.